src/routers/customCoursesRouter.ts

```diff
@@ -209,7 +289,36 @@ customCoursesRouter.get("/get", async (req: Request, res: Response) => {
     }
   } catch (error) {
     res.status(500).send(`Internal server error: ${error}`);
-  }
+  } 
+});
+
+customCoursesRouter.get("/getCustomCourses", async (req: Request, res: Response) => {
+
+  try{
+    const customCourses = await prisma.course.findMany({ 
+      where: {
+        OR: [
+          {owner: { role: "TUTOR" }},
+          {owner: { role: "PROFESSOR" }},
+        ]
+      },
+      include: {
+        lessons: true,
+      },
+    }); 
+
+    console.log("customCourses before if check", customCourses);
+
+    if (!customCourses) { 
```

Comment:
can we use `customCourses == null`?  or `customCourses != true`? This makes it more clear on the conditional statement